src/FixAcceptor.cpp

```diff
@@ -20,6 +21,13 @@
 using namespace FIX;
 using namespace std;
 
+FixAcceptor::FixAcceptor(FIX::SessionSettings settings, std::string storeFactory): FixConnection(settings, storeFactory) {
+	mAcceptor = new FIX::ThreadedSocketAcceptor(*mFixApplication, *mStoreFactory, mSettings, *mLogFactory);
+}
+
+FixAcceptor::~FixAcceptor() {
```

Comment:
Should this be `FixAcceptor:: FixAcceptor() {`? (minus the ~) As it looks now, it seems the default constructor is calling the destructor, no?